### PR TITLE
Doc updates for home page

### PR DESCRIPTION
**Pull request by emilyrohrbough**
### Summary
Some simple doc updates.

Resolves https://github.com/cerner/terra-toolkit/issues/83

### CODE REVIEW

**Review by bjankord — APPROVED** [review]
Nice!

File: README.md [hunk 1]
```diff
@@ -23,16 +26,33 @@ Terra Toolkit is a utility module used to facilitate independent development of
...
 
 ## Webdriver.io Utility
 
 [Webdriver.io](http://webdriver.io/) is a framework for writing webdriver powered tests to validate functionality in browsers. The Webdriver.io framework provides services for setting up a selenium server, starting webpack and static servers, running accessibility and visual regression testing, and more.
 
 See the [Webdriver.io Utility Developer's Guide](https://github.com/cerner/terra-toolkit/blob/master/docs/Wdio_Utility.md) to get started.
 
+## Webpack Configuration
+
+[Webpack](https://webpack.js.org/) is a module bundler used to compile modules with dependencies and generate static assets. Webpack is a very powerful tool that is highly configuration and Terra components rely on specific polyfills, webpack loaders and plugins to render correctly. Terra provides a [default webpack configuration](https://github.com/cerner/terra-toolkit/blob/master/config/webpack/webpack.config.js) which we recommend you extend to meet your needs. By using this default, we will manage webpack dependencies and set up translation aggregation.
```
Comment (yuderekyu) [inline README.md]: highly *configurable

File: docs/Webpack.md [hunk 2]
```diff
@@ -0,0 +1,75 @@
+# Webpack
+[Webpack](https://webpack.js.org/) is a module bundler used to compile modules with dependencies and generate static assets. Webpack is a very powerful tool that is highly configuration and Terra components rely on specific polyfills, webpack loaders and plugins to render correctly.
```
Comment (yuderekyu) [inline docs/Webpack.md]: highly *configurable

**Review by yuderekyu — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -23,16 +26,33 @@ Terra Toolkit is a utility module used to facilitate independent development of
 
 - Install with [npm](https://www.npmjs.com): `npm install terra-toolkit --save-dev`
 
-Terra Toolkit uses docker to setup, run, and tear down selenium to ensure a consistent testing environment locally and in continuous integration build systems. To use Terra Toolkit you must install docker on your machine. Installation instructions can be found at https://www.docker.com/.  **Requires Docker v17.09.0 or higher.**
+Terra Toolkit uses Docker to setup, run, and tear down selenium to ensure a consistent testing environment locally and in continuous integration build systems. To use Terra Toolkit for webdriver testing, you must install docker on your machine.
 
+- Install Docker version 17.09.0 or higher. Installation instructions can be found at https://www.docker.com/.
```
Comment (tomleewu) [inline README.md]: Docker has a nice installation guide: https://docs.docker.com/install/

It's kind of hard to find from the home page 👎

File: README.md [hunk 4]
```diff
@@ -23,16 +26,33 @@ Terra Toolkit is a utility module used to facilitate independent development of
...
 
+## Aggregate Translations Tool
+Terra components provide internationalization and localization support via `react-intl`. To use the component translations, the `aggregate-translations` pre-build tool will aggregate the translations, and create the intl loader and translation loader files that are configured for the specified locales.
+
+See the [Aggregating Translations Guide](https://github.com/cerner/terra-toolkit/blob/master/docs/AggregateTranslations.md) to get started.
+
+## Serve Options
+
+Terra Toolkit offers two ways to serve your client side application, `serve` and `serve-static`. Serve is a hot-reloading replacement for webpack-dev-server, which uses [webpack-serve](https://github.com/webpack-contrib/webpack-serve), while serve-static is a non-hot-reloading express server. Serve-static supports IE10 and is used in webdriver testing.
```
Comment (tomleewu) [inline README.md]: It might be helpful to have a link to what [Express](https://expressjs.com/) is

File: docs/Webpack.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+# Webpack
+[Webpack](https://webpack.js.org/) is a module bundler used to compile modules with dependencies and generate static assets. Webpack is a very powerful tool that is highly configurable and Terra components rely on specific polyfills, webpack loaders and plugins to render correctly.
+
+## Configuring Webpack
+Terra provides a [default webpack configuration](https://github.com/cerner/terra-toolkit/blob/master/config/webpack/webpack.config.js) via the `terra-toolkit` module which we recommend you extend to meet your needs. By using this default, we will manage webpack dependencies and set up translation aggregation. If you choose not to use the default configuration, it can be used as an guide to build your own.
```
Comment (tomleewu) [inline docs/Webpack.md]: > By using this default _configuration_, we will ...

File: README.md [hunk 6]
```diff
@@ -23,16 +26,33 @@ Terra Toolkit is a utility module used to facilitate independent development of
...
 
 ## Webdriver.io Utility
 
 [Webdriver.io](http://webdriver.io/) is a framework for writing webdriver powered tests to validate functionality in browsers. The Webdriver.io framework provides services for setting up a selenium server, starting webpack and static servers, running accessibility and visual regression testing, and more.
 
 See the [Webdriver.io Utility Developer's Guide](https://github.com/cerner/terra-toolkit/blob/master/docs/Wdio_Utility.md) to get started.
 
+## Webpack Configuration
+
+[Webpack](https://webpack.js.org/) is a module bundler used to compile modules with dependencies and generate static assets. Webpack is a very powerful tool that is highly configurable and Terra components rely on specific polyfills, webpack loaders and plugins to render correctly. Terra provides a [default webpack configuration](https://github.com/cerner/terra-toolkit/blob/master/config/webpack/webpack.config.js) which we recommend you extend to meet your needs. By using this default, we will manage webpack dependencies and set up translation aggregation.
```
Comment (tomleewu) [inline README.md]: > By using this default _configuration_, we will ...

File: docs/Webpack.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+### Extending the Default Config
+1. Create a `webpack.config.js` file.
+2. Import terra-toolkit's webpack configuration.
+2. Create an app-level webpack configuration which, at a minimum, supplies an entry and an [HtmlWebpackPlugin](https://github.com/jantimon/html-webpack-plugin) entry (version ^3.2.0 or higher).
+3. Use [`webpack-merge`](https://github.com/survivejs/webpack-merge) to combine the app config with terra-toolkit's default config. Note: since the default config is an function, it will need to be executed first.
+
+```javascript
```
Comment (tomleewu) [inline docs/Webpack.md]: Maybe add in an introduction to this code block,

_Here's an example of an app-level webpack configuration:_

File: docs/Webpack.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+
+const mergedConfig = (env, argv) => (
+  merge(defaultWebpackConfig(env, argv), appWebpackConfig(env, argv))
+);
+
+module.exports = mergedConfig;
+```
+
+#### Translation Aggregation
+Terra's supported locales will be aggregated when using the default webpack configuration through the `aggregate-translations` pre-build tool. To customize which translations are aggregated, refer these docs on [aggregating translations](https://github.com/cerner/terra-toolkit/blob/master/docs/AggregateTranslations.md#terrai18nconfig-example). Alternatively, translations are not desirable, disable translation aggregation within the webpack build by passing the environment variable `--env.disableAggregateTranslations` to the webpack command.
```
Comment (tomleewu) [inline docs/Webpack.md]: > Alternatively, _if_ translations are not _required_, disable translation ...

**Review by tomleewu — COMMENTED** [review]
This is super helpful 👍

**Review by dkasper-was-taken — APPROVED** [review]

**Review by BenBoersma — APPROVED** [review]

**Pull request merged**